app/status.go

```diff
@@ -453,6 +456,21 @@ func (a *App) GetCustomStatus(userID string) (*model.CustomStatus, *model.AppErr
 	return user.GetCustomStatus(), nil
 }
 
+func (a *App) GetRecentCustomStatuses(userID string) ([]*model.CustomStatus, error) {
+	pref, err := a.GetPreferenceByCategoryAndNameForUser(userID, model.PreferenceCategoryCustomStatus, model.PreferenceNameRecentCustomStatuses)
+
+	var existingRCS []*model.CustomStatus
```

Comment:
Why we are ignoring the error here? I mean the `err` variable from the `L:460`

Reply:
Handled and returned the `err`